### PR TITLE
add curation support for Maven and Pip

### PR DESCRIPTION
**Pull request by asafambar**

### CODE REVIEW

File: jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md [hunk 1]
```diff
@@ -11,27 +11,36 @@ JFrog Curation enables you to block malicious or risky open-source packages ente
 
 For more information on JFrog Curation and how to set it up, see the JFrog Curation general documentation at https://jfrog.com/help.
 
-JFrog Curation requires [Xray version 3.78.9](https://jfrog.com/help/r/jfrog-release-information/xray-3.78.9) and above, and [Artifactory version 7.63.5](https://jfrog.com/help/r/jfrog-release-information/artifactory-7.63) and above. It also requires Enterprise X and Enterprise + subscriptions.
-
+JFrog Curation requires [Xray version 3.78.9](https://jfrog.com/help/r/jfrog-release-information/xray-3.78.9) and above, and [Artifactory version 7.63.5](https://jfrog.com/help/r/jfrog-release-information/artifactory-7.63) and above. It also requires Enterprise X and Enterprise + subscriptions. 
+Currently, the following package managers are supported
+* Npm (npm)
+* Maven (mvn) - Requires xray 3.92 and above, and Artifactory 7.82 and above
```
Comment (attiasas) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md:17]:
```suggestion
JFrog Curation requires Enterprise X and Enterprise + subscriptions. 
Currently, the following package managers are supported:
* Npm (npm) - Requires [Xray version 3.78.9](https://jfrog.com/help/r/jfrog-release-information/xray-3.78.9) and above, and [Artifactory version 7.63.5](https://jfrog.com/help/r/jfrog-release-information/artifactory-7.63) and above
* Maven (mvn) - Requires [Xray version 3.92](https://jfrog.com/help/r/jfrog-release-information/xray-3.92) and above, and [Artifactory version 7.82](https://jfrog.com/help/r/jfrog-release-information/artifactory-7.82) and above
'''

File: jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md [hunk 2]
```diff
@@ -11,27 +11,36 @@ JFrog Curation enables you to block malicious or risky open-source packages ente
...
+1. Connect the JFrog CLI to your JFrog Platform instance by running the [**jf c add**](https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/configurations/jfrog-platform-configuration#adding-and-editing-configured-servers) command.
+2. Ensure your project is configured in the JFrog CLI with the repository you would like to resolve dependencies from. Here are details for each package manager:
+
+
+**NPM**
+* Set the repository with the [**jf npmc**](https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory#setting-npm-repositories) command inside the project directory</br>
+
+**MAVEN**
+* Set the repository with the [**jf mvnc**](https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory#setting-maven-repositories) command inside the project directory
+* Configure pass-through on the curated repositories
```
Comment (attiasas) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md:39]: Can you add a link to documentation about the `pass-through`? to help user configure it

**Review by attiasas — CHANGES_REQUESTED** [review]

File: jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md (hunk 2 above)
Comment (asafambar) [reply jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md:40]: yes, I'm waiting for our product to create a wiki page

File: jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md [hunk 3]
```diff
@@ -1,37 +1,64 @@
-# CLI for JFrog Curation
+# JFrog CLI curation-audit command (for JFrog Curation project audit)
```
Comment (attiasas) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md:1]:
```suggestion
# CLI for JFrog Curation
```
To match other titles

File: jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md [hunk 4]
```diff
@@ -1,37 +1,64 @@
...
-**Note**
+   Connect the JFrog CLI to your JFrog Platform instance by running the following command:
 
-> The **curation-audit** command currently only supports npm projects.
+    ```
+    jf c add
+    ```
 
-The **jf curation-audit** command enables developers to scan project dependencies to find packages that were blocked by the JFrog curation service. This command provides developers with more detailed information, such as whether the blocked package is the project’s direct dependency or is a transitive dependency. This information helps developers to resolve blocked packages more efficiently as they will be able to make a more informative decision based on what Policy violation occurred and what exactly needs to be resolved.
+   - When prompted for the access token, use the token generated from Artifactory. For more details, refer to the [JFrog CLI documentation](https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/configurations/jfrog-platform-configuration#adding-and-editing-configured-servers).
```
Comment (attiasas) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-curation.md:40]:
```suggestion
   - When prompted for the access token, use the token generated from Artifactory. For more details, refer to the [adding and editing configured servers documentation](https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/configurations/jfrog-platform-configuration#adding-and-editing-configured-servers).
```

**Review by attiasas — COMMENTED** [review]
make sure to fix conflicts

**Pull request merged**